django_cloud_tasks/tasks/task.py

```diff
@@ -319,18 +319,23 @@ def discard(cls, task_id: str | None = None, min_retries: int = 0):
         else:
             task_objects = client.list_tasks(queue_name=cls.queue())
 
-        outputs = []
-        for task_obj in task_objects:
+        def process_task(task_obj, min_retries, cls, client):
```

Comment:
Is `cls` really needed? Because it's accessible from this inner method.

Reply:
I was worried about avoiding potential concurrency issues. But I think is not really needed, I feel confident in removing that